File: 724-find-pivot-index/find-pivot-index.cpp
class Solution {
public:
    int pivotIndex(vector<int>& nums) 
    {
        int l = 0;
        int r = 0;
        int sum = 0;
        for(int i = 0;i < nums.size();i++)
        {
            sum += nums[i];
        }
        for(int i = 0;i < nums.size();i++)
        {
            int p = nums[i];
            r = sum - l - p;
            if(r == l)
            {
                return i;
                break;
            }
            else
            {
                l += p;
            }
        }
       return -1;
    }
};

File: 724-find-pivot-index/find-pivot-index.py
class Solution(object):
    def pivotIndex(self, nums):
        n = len(nums)
        tot = sum(nums)
        left = 0
        for i in range(n):
            if(left == tot - left - nums[i]):
                return i
            left += nums[i]
        return -1

        